imagebuildah/chroot_symlink.go

```diff
@@ -169,7 +176,12 @@ func modTimeIsGreater(rootdir, path string, historyTime string) (bool, error) {
 		}
 		return nil
 	})
+
 	if err != nil {
+		// if error is due to dangling symlink, ignore error and return nil
+		if err == errDanglingSymlink {
```

Comment:
... and this will need to compare with `errors.Cause(err)`.

Reply:
Fixed